### ISSUE TITLE
Version 0.14.14 - Metrics could not found: moleculer.request.error.total

### ISSUE THREAD

**Issue by kendev9x** [moleculerjs/moleculer#960]
Hi all, i am using lasted version Moleculer. I read at document metric have a property is: moleculer.request.error.total and fews about report errors.

But when i access metrics on web browser. I can not found them.

Please help me. Thanks you

**Comment by icebob** [comment]
Could you explain? Which site and where...etc

**Comment by kendev9x** [comment]
Thank you you. I just only run moleculer project at local.

Then i try to changes Metrics with:

metrics: {
    enabled: true,
    reporter: [
      {
        type: "Prometheus",
        options: {
          port: 9001,
          path: "/metrics",
          defaultLabels: registry => ({
            namespace: registry.broker.namespace,
            nodeID: registry.broker.nodeID
          })
        }
      }
    ]
  },
  
  Then i access via: http://127.0.0.1:9001/metrics and i could not found property is: moleculer.request.error.total.

**Comment by icebob** [comment]
It will appear if there is any error.

**Comment by kendev9x** [comment]
Thanks you so much. Let me try again. I aldready think it will default. If have not error so it will have a value as 0
